### PR TITLE
 Allow list of fonts in font family property

### PR DESCRIPTION
**Pull request by claudep**
Here's a slightly different approach of the PR #122.

### CODE REVIEW

File: svglib/svglib.py [hunk 1]
```diff
@@ -53,27 +53,33 @@
...
 
 logger = logging.getLogger(__name__)
 
 Box = namedtuple('Box', ['x', 'y', 'width', 'height'])
 
 
 def find_font(font_name):
-    if font_name in STANDARD_FONT_NAMES or font_name in _registered_fonts:
-        return font_name
+    """Return the font and a boolean indicating if the match is exact."""
```
Comment (deeplook) [inline svglib/svglib.py]: "Boolean" or "bool" (sorry for nit-picking :)

Comment (claudep) [reply svglib/svglib.py]: Thanks, I learned something today :smiley:

**Comment by replabrobin** [comment]
Hi Claude, this works for me and my windows test do pass.

**Comment by claudep** [comment]
Thanks Robin for initiating this feature!

**Comment by replabrobin** [comment]
I have updated my svglib repo to address issue 117 (the ellipse zerodiv error), pr 114. I needed the latter as part of code to do backfilling of clip-paths as I had an example where the def of the clip path was at the end of the file but was defining a rect.

How should I go about presenting this code to you? The clip-path backfill seems to work well.

**Comment by claudep** [comment]
You may review #114 and #118 so as we can merge them in master soon. I'm not sure about what code you want to present, is this svglib code? Feel free to point us to some branch of yours with this code.

**Comment by replabrobin** [comment]
This is pure svglib; I have an example where the clip-path is used several times as 
<path clip-path="url(#p38b2f8081d)" d="M -54.609375 320.4...../>

and later we see 
<defs>
  <clipPath id="p38b2f8081d">
   <rect height="277.2" width="697.5" x="112.5" y="43.2"/>
  </clipPath>
</defs>

this fails to clip in latest svglib; so I have made changes to make this work.

The result is in master https://github.com/replabrobin/svglib

**Comment by deeplook** [comment]
👍  Thanks, @replabrobin and @claudep!

**Pull request merged**